tests/exporters/exporters_utils.py

```diff
@@ -176,6 +176,7 @@
     "roformer": "junnyu/roformer_chinese_base",
     "segformer": "nvidia/segformer-b0-finetuned-ade-512-512",
     "squeezebert": "squeezebert/squeezebert-uncased",
+    "splinter": "hf-internal-testing/tiny-random-SplinterModel",
```

Comment:
Can you add it in `PYTORCH_EXPORT_MODELS_TINY` as well? It's actually this dict that is used for testing.

Reply:
Yes you're right, my bad for not adding it there as well. Thank you for your feedback !